config/cache.php

```diff
@@ -73,6 +73,12 @@
             'connection' => 'cache',
         ],
 
+        'session' => [
+            'driver' => env('SESSION_DRIVER', 'array'),
```

Comment:
@wjgilmore just have updated to use env instead of static val.
I just reused the existing `SESSION_DRIVER` var to not to create another one since this config should always be the same as `SESSION_DRIVER` one.